LINUX_MACOS_SETUP.md

```diff
@@ -0,0 +1,111 @@
+# Linux/MacOS Setup
+
+This setup guide uses Docker. If you have Linux and want to proceed with the manual setup instead, please take a look at `MANUAL_SETUP.md` for more details.
+
+## Installation
+
+Before proceeding, make sure `python3` and `git` are installed.
+
+### 1. Clone the repository
+
+Using **SSH**:
+
+```sh
+git clone git@github.com:utat-ss/finch-flight-software.git
+cd finch-flight-software
+```
+
+Or using **HTTPS**:
+
+```sh
+git clone https://github.com/utat-ss/finch-flight-software.git
+cd finch-flight-software
+```
+
+### 2. Docker
+For Linux, use [this](https://docs.docker.com/engine/install/) guide.
+
+For MacOS, install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+### 3. Pyocd
+
+In your terminal, run
+```
+python3 -m pip install -U pyocd
+```
+
+This command should work fine on Linux x86 and Macs.
+
+If you have a Raspberry Pi or some other non-standard platform to install pyocd on, check out [this](https://pyocd.io/docs/installing_on_non_x86) guide for more information.
+
+### 4. Pyocd packs
+
+Pyocd needs Keil packs to flash the code to stm32 boards.
+
+#### 4.0. Create a folder
+
+* Create folder named `packs` in the project root.
+
+#### 4.1. Download pyocd pack for stm32h7
+
+* Open [this](https://www.keil.arm.com/packs/stm32h7xx_dfp-keil/overview/) page or search `stm32h7 pack keil`.
+* Download the .pack file and place it in the `packs` folder.
+* Rename the .pack file to `h7.pack`.
+
+#### 4.2. Download pyocd pack for stm32g4
+
+* Open [this](https://www.keil.arm.com/packs/stm32g4xx_dfp-keil/overview/) page or search `stm32g4 pack keil`.
+* Download the .pack file and place it in the `packs` folder.
+* Rename the .pack file to `g4.pack`.
+
```

Comment:
I think there is a CLI way of doing this (something like `pyocd pack install ...`). I've done it before, but I'm not on the same machine right now, so I can't check my `zsh_history`. Is there any reason to manually go to the website and get the `.pack` file?